### ISSUE TITLE
Permission denied while the /dist exists

### ISSUE THREAD

**Issue by AntonDnepr** [ancoopa/static-pages-bundler#1]
Env:
xubuntu 18.04
node v8.11.4

Structure:
```
drwxrwxr-x  2 anton anton  4096 кві 22 18:10 dist/
-rw-rw-r--  1 anton anton 16789 кві 22 18:06 index.html
```

Error
```
/home/anton/.nvm/versions/node/v8.11.4/lib/node_modules/static-pages-bundler/bundler/modules/fileManager.js:56
        throw err;
        ^

Error: EACCES: permission denied, mkdir '/dist'
    at Object.fs.mkdirSync (fs.js:885:18)
    at FileManager.checkCreateDirsForFile (/home/anton/.nvm/versions/node/v8.11.4/lib/node_modules/static-pages-bundler/bundler/modules/fileManager.js:53:14)
    at FileManager.createWriteFile (/home/anton/.nvm/versions/node/v8.11.4/lib/node_modules/static-pages-bundler/bundler/modules/fileManager.js:26:10)
    at Bundler.processBundleUnit (/home/anton/.nvm/versions/node/v8.11.4/lib/node_modules/static-pages-bundler/bundler/index.js:66:29)
    at Bundler.createBundle (/home/anton/.nvm/versions/node/v8.11.4/lib/node_modules/static-pages-bundler/bundler/index.js:21:10)
    at Object.<anonymous> (/home/anton/.nvm/versions/node/v8.11.4/lib/node_modules/static-pages-bundler/index.js:22:9)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)

```

**Comment by ancoopa** [comment]
Thank you for the report, Anton.
The issue fixed in a fresh update (1.0.21).
Please, update the library to apply the changes (`$ npm i static-pages-bundler` or `$ npm i -g static-pages-bundler`) and try to use it again.
I apologize for the inconvenience.

**Comment by ancoopa** [comment]
Thank you for the report, Anton.
The issue fixed in a fresh update (1.0.21).
Please, update the library to apply the changes (`$ npm i static-pages-bundler` or `$ npm i -g static-pages-bundler`) and try to use it again.
I apologize for the inconvenience.
Please, let me know if the issue is still there.
